lib/runtime/goto.js

```diff
@@ -1,17 +1,44 @@
 /** @babel */
 
+import path from 'path'
+import { Point, Range } from 'atom'
+
 import { client } from '../connection'
 import modules from './modules'
 import { isValidScopeToInspect } from '../misc/scopes'
-import { getWord, isValidWordToInspect, wordRegex } from '../misc/words'
+import { getWord, isValidWordToInspect } from '../misc/words'
 
 const methods = client.import('methods')
 
+const includeRegex = /(include|include_dependency)\(".+\.jl"\)/
+const filePathRegex = /".+\.jl"/
+
 class Goto {
   activate (ink) {
     this.ink = ink
   }
 
+  getJumpFilePath(editor, bufferPosition) {
+    const { row, column } = bufferPosition
+    const text = editor.lineTextForBufferRow(row)
+    let match = text.match(includeRegex)
+
+    if (!match) return false
+
+    match = text.match(filePathRegex)
+    const filePathBody = match[0].replace(/"/g, '')
+    const dirPath = path.dirname(editor.getPath())
+    const filePath = path.join(dirPath, filePathBody)
+    const columnStart = match.index + 1
+    const columnEnd = columnStart + filePathBody.length
+    if (columnStart <= column && column <= columnEnd) {
```

Comment:
I think we need to check if `filePath` actually is a file somewhere, otherwise you'll end up in an empty editor, which might be confusing.

Reply:
I intentionally didn't add the check, assuming it would be also used for file creation, but does this sound weird to you ?

Reply:
Umm yeah, creating a new file is definitely not a `goto/jump` stuff.
Will address this.

Reply:
f7afd54 :)